mix.exs

```diff
@@ -40,11 +40,10 @@ defmodule GRPC.Mixfile do
   defp deps do
     [
       {:cowboy, "~> 2.9"},
-      # This is the same as :gun 2.0.0-rc.2,
-      # but we can't depend on an RC for releases
-      {:gun, "~> 2.0.1", hex: :grpc_gun},
+      {:gun, "~> 2.0"},
       {:mint, "~> 1.4.2"},
-      {:cowlib, "~> 2.11"},
+      # This is temporal only for code review until https://github.com/ninenines/cowboy/pull/1601 is merged
```

Comment:
Please, do not merge until `cowboy` is released, this is for the sake of the PR review.

Reply:
We can't actually release anything with `override: true` in Hex, so I'll leave a "request changes" on this :)

Reply:
Sure! This change is on purpose for the shake of code review until the depending library is released. Then, we just need to change this. As you said this is a request change :)